### ISSUE TITLE
LUNA16 数据预处理

### ISSUE THREAD

**Issue by Pang-b0** [mj129/SANet#21]
Hello, author, I would like to ask you how SANet reproduces the results on luna16 data set, how to realize the data preprocessing part, and can you provide the preprocessed data or code including the training set verification set and the test set?
